### PR TITLE
Feature/update to coral backend

### PR DESCRIPTION
**Pull request by dutsuwak**
* Updates the media_server script to use Coral backend

### CODE REVIEW

File: media-server/config.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 CAMERA_DEVICE_WIDTH=1280
 CAMERA_DEVICE_HEIGHT=720
-MODEL_LOCATION=/home/fsolano/work/devdirs/projects/GoogleCoral/coral-gst-inference-demos/media-server/graph_tinyyolov3_tensorflow.pb # Use absolute path.
-INPUT_LAYER='inputs'
-OUTPUT_LAYER='output_boxes'
-LABELS='labels.txt'
+MODEL_LOCATION=/home/fsolano/work/devdirs/projects/GoogleCoral/coral-gst-inference-demos/media-server/mobilenet_v2_1.0_224_quant_edgetpu.tflite # Use absolute path.
+INPUT_LAYER='input'
+OUTPUT_LAYER='MobilenetV2/Predictions/Reshape_1'
+LABELS='imagenet_labels.txt'
```
Comment (michaelgruner) [inline media-server/config.txt]: Specify here that this file needs to change according to the training of the different models.

File: media-server/config.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 OUTPUT_FILE='/home/fsolano/work/devdirs/projects/GoogleCoral/coral-gst-inference-demos/media-server/recording.mp4' # Only mp4 is supported by demo. Use absolute path.
```
Comment (michaelgruner) [inline media-server/config.txt]: I think this will fail if the directories don't exist. Please use a more standard location.

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+By using GstInference as the interface between Coral and GStreamer, users can:
+
+* Easily prototype GStreamer pipelines with common and basic GStreamer tools such as gst-launch and GStreamer Daemon.
+* Easily test and benchmark TFLite models using GStreamer with Coral.
+* Enable a world of possibilities to use Coral with video feeds from cameras, video files, and network streams, and process the prediction information (detection, classification, estimation, segmentation) to monitor events and trigger actions.
+* Develop intelligent media servers with recording, streaming, capture, playback, and display features.
+* Abstract GStreamer complexity in terms of buffers and events handling.
+* Abstract TensorFlow Lite complexity and configuration.
+* Make use of GstInference helper elements and API to visualize and easily extract readable prediction information.
+
```
Comment (rrcarlosrodriguez) [inline README.md:16]: Please add the following:
- GstInference logo
- Link to GstInference
- Link to Coral wiki page

**Review by rrcarlosrodriguez — CHANGES_REQUESTED** [review]

**Review by michaelgruner — CHANGES_REQUESTED** [review]

**Review by michaelgruner — APPROVED** [review]
approved with comments

**Pull request merged**